types/rethinkdb/index.d.ts

```diff
@@ -221,6 +224,18 @@ declare module "rethinkdb" {
          * See: https://rethinkdb.com/api/javascript/has_fields/
          */
         hasFields(...fields: string[]): T;
+    } 
+
+    interface Geometry {
```

Comment:
TypeScript uses a structural type system. An empty interface is equivalent to `{}`.